cluster/examples/workloads/kubernetes/wordpress-aws/provider.yaml

```diff
@@ -33,6 +33,7 @@ parameters:
 provisioner: rdsinstance.database.aws.crossplane.io/v1alpha1
 providerRef:
   name: aws-provider
+  namespace: crossplane-system
```

Comment:
I know this PR is focused on AWS, but since some of the GCP example files are also being touched, do you want to make this update on `wordpress-gcp/provider.yaml` as well?

Reply:
the `wordpress-gcp/provider.yaml` already has the namespace mentioned

Reply:
Ah totally missed that, my apologies!